src/backend/renderer/pixman/mod.rs

```diff
@@ -41,30 +42,26 @@ mod error;
 
 pub use error::*;
 
-lazy_static::lazy_static! {
-    static ref SUPPORTED_FORMATS: Vec<drm_fourcc::DrmFourcc> = {
-        vec![
-            #[cfg(target_endian = "little")]
-            DrmFourcc::Rgb565,
-            DrmFourcc::Xrgb8888,
-            DrmFourcc::Argb8888,
-            DrmFourcc::Xbgr8888,
-            DrmFourcc::Abgr8888,
-            DrmFourcc::Rgbx8888,
-            DrmFourcc::Rgba8888,
-            DrmFourcc::Bgrx8888,
-            DrmFourcc::Bgra8888,
-            #[cfg(target_endian = "little")]
-            DrmFourcc::Xrgb2101010,
-            #[cfg(target_endian = "little")]
-            DrmFourcc::Argb2101010,
-            #[cfg(target_endian = "little")]
-            DrmFourcc::Xbgr2101010,
-            #[cfg(target_endian = "little")]
-            DrmFourcc::Abgr2101010,
-        ]
-    };
-}
+const SUPPORTED_FORMATS: [DrmFourcc; 13] = [
+    #[cfg(target_endian = "little")]
+    DrmFourcc::Rgb565,
+    DrmFourcc::Xrgb8888,
+    DrmFourcc::Argb8888,
+    DrmFourcc::Xbgr8888,
+    DrmFourcc::Abgr8888,
+    DrmFourcc::Rgbx8888,
+    DrmFourcc::Rgba8888,
+    DrmFourcc::Bgrx8888,
+    DrmFourcc::Bgra8888,
+    #[cfg(target_endian = "little")]
+    DrmFourcc::Xrgb2101010,
+    #[cfg(target_endian = "little")]
+    DrmFourcc::Argb2101010,
+    #[cfg(target_endian = "little")]
+    DrmFourcc::Xbgr2101010,
+    #[cfg(target_endian = "little")]
+    DrmFourcc::Abgr2101010,
+];
```

Comment:
This does not seem right, how can there be always 13 elements, if 5 of the elements are cfg guarded?

Reply:
Thanks! I'm not completely familiar with Rust yet. Should Vec still be used here because static/const must specify the type? (Or is there a better way?)

Reply:
I've fixed by using a `cfg(not())`